tests/integration/connection_manager_translate_test.py

```diff
@@ -0,0 +1,137 @@
+import uuid
```

Comment:
Related to tests in general. Did you understand what testTranslateFromMemberIsNotUsedWhenAlreadyTranslatedByAddressProvider test does in java side? I am confused with the test and also I see that you don't include it. Was it hard for you to get as well?

Reply:
Yes, basically, if you set the property to true and use a custom address provider, we will ignore the logic of the address translation for members altogether. So, the test seems weird to me